assets/sass/templates/text-input-single-line.hbs

```diff
@@ -5,7 +5,7 @@
   </p>
   <p>
     <label for="numberin">A <code>number</code> input field</label>
-    <input name="numberin" id="numberin" type="number" pattern="[0-9]*" />
```

Comment:
Do we need to remove the ref to "pattern="[0-9]*" in our documentation?

We should also test whether this removes iOS support for the number pad?